### ISSUE TITLE
Mouse Scrollbar Unresponsive

### ISSUE THREAD

**Issue by mbway** [neovide/neovide#2226]
**Describe the bug**

Mouse events for [`nvim-scrollview`](https://github.com/dstein64/nvim-scrollview) do not work well in neovide.

In the terminal and neovim-qt, the scrollbar can be dragged with the mouse and is very responsive.

Up to neovide version 0.11.2, using the mouse to move the scrollbar does not update consistently.

Since neovide version 0.12.0 the scrollbar no longer responds to the mouse at all. This is probably related to https://github.com/neovide/neovide/issues/2222 but there were issues with the behaviour before as well.

**To Reproduce**

1. use this minimal `init.lua` (assuming lazy vim already installed)
```lua
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
vim.opt.rtp:prepend(lazypath)
vim.opt.number = true -- to show the view scrolling more easily
require("lazy").setup({
    {
        "dstein64/nvim-scrollview",
        config = function()
            require('scrollview').setup({})
        end
    },
})
```
2. run `neovide -- -u init.lua <large_file>`
3. attempt to use the mouse to move the scrollbar
4. run `nvim -u init.lua <large_file>`
5. attempt to use the mouse to move the scrollbar

**Expected behavior**
The scrollbar should respond to mouse input like in the terminal and other GUIs like neovim-qt

**Screenshots**

https://github.com/neovide/neovide/assets/4923501/f1c8dba5-4784-4b77-9326-86b8779bf6cd

**Desktop (please complete the following information):**
 - OS: Arch Linux, KDE Plasma 5, X11
 - Neovide Version 0.12.0
 - Neovim Version 0.9.4

**Comment by fredizzimo** [comment]
It wonder why that plugin don’t use `<LeftDrag>`, which would work

https://neovim.io/doc/user/term.html#%3CLeftDrag%3E

But for now, I guess you need to set `mousemoveevent` to true.

Making it more responsive is tricky, because the library we are using `nvim-rs` does not support notifications, so we are forced to wait for a reponse from Neovim, before we can send the next event.  https://github.com/KillTheMule/nvim-rs/issues/7

**Comment by mbway** [comment]
> But for now, I guess you need to set mousemoveevent to true

if you mean adding `vim.opt.mousemoveevent = true` to the init.lua, that seems to have no effect. 

nor does setting `vim.keymap.set({'n', 'v', 'i'}, '<LeftDrag>', '<plug>(ScrollViewLeftMouse)')` but perhaps it's more complicated than that.

**Comment by fredizzimo** [comment]
Yes, sorry I thought that would bring it back to the same state as previous versions. But if it isn’t then it’s probably this one that caused it
* https://github.com/neovide/neovide/pull/2082

Which is bad news, since I don’t see a way to fix both bugs, and we probably need a Neovim fix.

Regarding `LeftDrag`, my point is that the plugin itself should use that. As a user, that’s not something you can do yourself.

**Comment by fredizzimo** [comment]
This seems to be caused by `--multigrid` now being the default, and furthermore a Neovim bug reported here:
* https://github.com/neovim/neovim/issues/27181

**Comment by fredizzimo** [comment]
The workaround for now, is to use `neovide --no-multigrid`

**Comment by fredizzimo** [comment]
There's a fix on the way now https://github.com/neovide/neovide/pull/2919, but there might be some troubles geting things merged on the Neovim side.
